src/components/MdTable/MdTable.vue

```diff
@@ -153,7 +158,9 @@
           getModel: this.getModel,
           getModelItem: this.getModelItem,
           selectingMode: null
-        }
+        },
+        items: this.value || [],
```

Comment:
I think this should be as Computed

Reply:
Hmm I think I'll disagree with this one. This is because we'll have to set the value once the Promise resolves.
Vue recommends not modifying props directly and instead using a data property, or computed if possible. Since there's a set involved, shouldn't data be the more natural choice?

Reply:
Yes but when i pass my array it will be not reactive to changes because data are changed only on mounted. 

And there are few things that you forgot to change `2 methods (getModel, getModelItem) that are still pointing to old this.value`

Reply:
Hmm good point. I believe this can be resolved by
```
watch: {
  value() {}
}
```
as well. Which do you prefer?

Reply:
Yeah, watch should be good